pkg/sources/filesystem/filesystem.go

```diff
@@ -93,7 +96,7 @@ func (s *Source) Chunks(ctx context.Context, chunksChan chan *sources.Chunk, _ .
 		fileInfo, err := os.Stat(cleanPath)
 		if err != nil {
 			logger.Error(err, "unable to get file info")
-			continue
+			return nil
```

Comment:
This seems like a really big change! Is it related to concurrency? Should it be in this PR?

Reply:
Whoops this was left behind when we had first put the concurrency within this level. reverted it.